js/circuit-breaker-callback.js

```diff
@@ -14,6 +14,9 @@
     var checkBalanceURL = "https://global-ebank.openliberty.io/checkBalance";
     var welcomePageURL = "https://global-ebank.openliberty.io/welcome";
     var isRefreshing = false;
+    var mapStepNameToScollLine = { 'ConfigureFailureThresholdParams': 14, 
+                                   'ConfigureDelayParams': 16,
+                                   'ConfigureSuccessThresholdParams': 16 };
```

Comment:
I am wondering why the 2 line numbers to scroll to are not the same as that used in __addCircuitBreakerAnnotation function. The point is to have this new property table so that __addCircuitBreakerAnnotation will use it too.